### ISSUE TITLE
HTML syntax colour wrong after html comment

### ISSUE THREAD

**Issue by BillyFB** [lapce/lapce#1687]
## Lapce Version

v0.2.3

## System information

Windows 11 22H2

## Describe the bug
HTML syntax colour wrong or ignored after html comment.

## Additional information

![image](https://user-images.githubusercontent.com/1252105/201340896-b8d52bf4-d8ae-4e6b-b44b-b854ab960d7f.png)

**Comment by micha-b** [comment]
For HTML language files there is no comment token set at the [HTML SyntaxProperties](https://github.com/lapce/lapce/blob/c355bb2fe23d4af587d3122f06bccc2b00fca7c4/lapce-core/src/language.rs#L453). 

I do not know if it is just missing or if it would cause problems if set there (with "<!--") because HTML only has opening- and closing-comments and no line-comments. 
The state of SyntaxProperties for most other languages currently only lists their line-comment tokens.

But I could also be misunderstanding the problem because I am quite new to the project and this is my first comment here :)
So do feel free to correct me if I am wrong!

**Comment by RenjiSann** [comment]
The token comment is set to `//` for the [XML language](https://github.com/lapce/lapce/blob/c355bb2fe23d4af587d3122f06bccc2b00fca7c4/lapce-core/src/language.rs#L858), which is working as intended. Maybe doing the same will fix this issue ?

**Comment by panekj** [comment]
XML and HTML are different languages with different syntax highlighting

**Comment by RenjiSann** [comment]
Yeah, I worked on the XML tree-sitter. I don't remember why I added `//` to the comment element though, and thought that it might work by adding it. I've just tried and it doesn't fix anything. Could the problem come from the `tree-sitter-html` dependency ? I find it surprising as it is an *official* crate from tree-sitter.

**Comment by MinusGix** [comment]
I believe the comment field in `SyntaxProperties` is purely for toggling whether a line is commented with a keybind (though it needs some work for making it work with multiline comments), so it isn't the cause of the syntax highlighting issues.  
So, this is probably some issue in our tree-sitter code or the queries we use (but Helix uses relatively similar queries and theirs works fine, so probably our tree-sitter code).

**Comment by salvoc81** [comment]
Just wanted to confirm that I am having the same issue on v0.2.4

**Comment by antonyadhiban** [comment]
Facing same issue. Fixing this would complete my migration from vscode to lapce.
